desktop-src/direct3dhlsl/AddUint64.md

```diff
@@ -0,0 +1,61 @@
+---
+description: Unsigned add of 32-bit operand with the carry.
+nms.assetid:
+title: AddUint64
+ms.topic: reference
+ms.date: 07/10/2024
+topic_type:
+- APIRef
+- kbSyntax
+api_name:
+- AddUint64
+api_type:
+- NA
+---
+
+
+# AddUint64
+
+Unsigned add of 32-bit operand with the carry.
```

Comment:
This description is pretty unclear. Probably just have to write one up manually.